content/_changelogs/11.0.0.md

```diff
@@ -1,17 +1,105 @@
+  `stylesheets` from the `options` argument to the `cy.mount()` function. We
+  recommend writing test-specific styles in a separate `css` file you import in
+  your test, or in your `supportFile`. Addresses
+  [#24328](https://github.com/cypress-io/cypress/issues/24328)
+- Subsequent `cy.mount` calls within the same test will remove the last mounted
+  component from the DOM. Addresses
+  [#24329](https://github.com/cypress-io/cypress/issues/24329).
+- `mountHook` from `cypress/react` has been removed. We recommend replacing it
+  with `mount` and a component. Addresses
+  [#24328](https://github.com/cypress-io/cypress/issues/24328)
+- `unmount` from `cypress/react` has been removed. We recommend using the API
+  React provides for unmounting components,
+  [unmountComponentAtNode](https://reactjs.org/docs/react-dom.html#unmountcomponentatnode).
+  Addresses [#24328](https://github.com/cypress-io/cypress/issues/24328)
+- `mountCallback` from `cypress/vue` has been removed. We recommend using
+  `mount`. Addresses
+  [#24328](https://github.com/cypress-io/cypress/issues/24328)
+- `mount` from `cypress/vue` now returns an object with both the VueWrapper
+  (wrapper) and the component instance (component). Addresses
+  [#24342](https://github.com/cypress-io/cypress/issues/24342)
+- When providing an inline `viteConfig` inside of `cypress.config`, any
+  `vite.config.js` file is not automatically merged. Addresses
+  [#24133](https://github.com/cypress-io/cypress/issues/24133)
+- Angular providers passed as part of the mounting options will be assigned at
+  the module level using the `TestBed.configureTestingModule` API. This means
+  that module-level providers (resolved from imports or
+  `@Injectable({ providedIn: 'root' })` can be overridden, but providers
+  specified in `@Component({ providers: [...] })` will not be overridden when
+  using `cy.mount(MyComponent, { providers: [...] })`. To override
+  component-level providers, use the `TestBed.overrideComponent` API. Addresses
+  [#24047](https://github.com/cypress-io/cypress/issues/24047) and
+  [#23427](https://github.com/cypress-io/cypress/issues/23427)
+
+**Experimental Breaking Changes:**
+
+- [`cy.session()`](/api/commands/session) now requires a setup command to use
+  the command. Addresses
+  [#24311](https://github.com/cypress-io/cypress/issues/24311).
 
 **Features:**
 
-**Bugfixes:**
+- Incorporated V8 snapshots into the build process of the Electron binary to
+  improve startup time and reduce the time to download and unzip the binary
+  during installation. Addresses
+  [#18480](https://github.com/cypress-io/cypress/issues/18480).
+- `cy.getCookie()`, `cy.getCookies()`, `cy.setCookie()`, `cy.clearCookie()`, and
+  `cy.clearCookies()` now accept a domain option. Addresses
+  [#24264](https://github.com/cypress-io/cypress/issues/24264)
+- Communication with the Cypress Dashboard will now verify CAs and reject any
```

Comment:
Is this not considered breaking?

Reply:
My initial thought was yes as well, but I think the argument for it not being breaking was that people in this situation already had to have one of these variables set for things to work properly (e.g. to download the binary)

Reply:
Moved to breaking. [8526a30](https://github.com/cypress-io/cypress-documentation/pull/4844/commits/8526a30a7a76872a43bd1d18a66315b229e0b126)